spec/spec_helper.rb

```diff
@@ -11,6 +11,7 @@
 require "byebug" if ENV["DEBUG"]
 
 require "dynamodb_local"
+require "new_class_helper"
```

Comment:
See line 37/38 below:
```
 Dir["#{File.dirname(__FILE__)}/support/**/*.rb"].each {|f| require f}
```
All we have to do is put support files in `spec/support` and they will be required.  Please move the `new_class_helper` there.

Reply:
@pboling done